superset/views/alerts.py

```diff
@@ -248,5 +170,12 @@ def pre_add(self, item: "AlertModelView") -> None:
         if not croniter.is_valid(item.crontab):
             raise SupersetException("Invalid crontab format")
 
+        item.validator_type = item.validator_type.lower()
+        check_validator(item.validator_type, item.validator_config)
+
+    def pre_update(self, item: "AlertModelView") -> None:
+        item.validator_type = item.validator_type.lower()
+        check_validator(item.validator_type, item.validator_config)
```

Comment:
It brings a better used experience to use Flask-WTF validators instead: https://wtforms.readthedocs.io/en/2.3.x/validators/#module-wtforms.validators. This way the user get's alerted about a validation fail while still on the form and not after submitting

Reply:
this code will be removed in a couple of weeks as I understand, will keep it this way for now.